stylus/ui-app/alerts.styl

```diff
@@ -45,17 +52,20 @@ $alerts
         .coz-alert-title
             font-weight bold
 
-    .coz-alert--error
-        background-color pomegranate
-
-    &.coz-alerter--hidden
```

Comment:
I don't get the point of removing this rule, for me it allows us to toggle visibility on both `.coz-alert` and `.coz-overlay`at the same time, while triggering transitions. We toggle the`*--hidden` class on one element instead of two (or maybe more in the future).